src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java

```diff
@@ -28,7 +28,7 @@ public Optional<User> findByEmail(String email) {
     }
 
     @Override
-    public User get(Long id) {
+    public User get(java.lang.Long id) {
```

Comment:
some are still present, just look at the changed files on git like you usually review other people's code, or search by the whole project in IDEA something like `lang.Long ` and you should be able to find all of them